python/mxnet/ndarray/numpy_extension/random.py

```diff
@@ -0,0 +1,101 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Namespace for operators used in Gluon dispatched by F=ndarray."""
+from __future__ import absolute_import
+from ...context import current_context
+from ..numpy import _internal as _npi
+
+
+__all__ = ['bernoulli']
+
+
+def bernoulli(prob, logit, size, dtype, ctx, out):
+    """Creates a Bernoulli distribution parameterized by :attr:`prob`
+    or :attr:`logit` (but not both).
+
+    Samples are binary (0 or 1). They take the value `1` with probability `p`
+    and `0` with probability `1 - p`.
+
+    Parameters
+    ----------
+    prob : float, ndarray
+        The probability of sampling '1'.
+    logit : float, ndarray
+        The log-odds of sampling '1'.
```

Comment:
We should clarify that if we should either set `logtis` or `prob`. If we set `logtis`, the prob is `sigmoid(logits)`

Reply:
You are right, I shall also add information regarding `ValueError` into the docstring.

Reply:
You can also refer to the TF: https://www.tensorflow.org/probability/api_docs/python/tfp/distributions/Bernoulli

Reply:
> We should clarify that if we should either set `logtis` or `prob`. If we set `logtis`, the prob is `sigmoid(logits)`

I think this is little bit redundant, the term `log-odds` already explains the relation between  logit and prob. Also, the docstring is copied from torch.distribution.